Cargo.toml

```diff
@@ -75,14 +75,17 @@ description = "Zenoh: Zero Overhead Pub/sub, Store/Query and Compute."
 # DEFAULT-FEATURES NOTE: Be careful with default-features and additivity!
 #                        (https://github.com/rust-lang/cargo/issues/11329)
 [workspace.dependencies]
+
+
+async-rustls = "0.4.0"
```

Comment:
Why reintroducing `async-rustls` while the implementation moved to `tokio-tls`?

Reply:
Good catch. It doesn't seem to be used in the code, and it is not present in the lockfile.